### DISCUSSION TITLE
Bind an action to pressing SUPER key only

### DISCUSSION THREAD

**Discussion by LOSEARDES77** [hyprwm/Hyprland#2506] [Q&A] [answered]
i just wanted to know if there is any way to bind something to the SUPER key alone

**Comment by vaxerski** [comment, marked as answer]
`bindr=SUPER,Super_L,exec,amongus`

**Comment by donovanglover** [comment]
You can also use [dual-function-keys](https://gitlab.com/interception/linux/plugins/dual-function-keys), which will only execute the keybind if no other key combinations were used. Here's an [example](https://github.com/donovanglover/nix-config/commit/49a621f081191497494b57c8c5a92c68fea2e845) from my config.
